src/cow-react/pages/LimitOrders/index.tsx

```diff
@@ -5,24 +5,29 @@ import {
   InfoPopup,
   MarketPriceUpdater,
   ActiveRateUpdater,
+  OrdersWidget,
+  limitOrdersAtom,
 } from '@cow/modules/limitOrders'
+import { useAtomValue } from 'jotai/utils'
+
 export default function LimitOrderPage() {
+  const { isUnlocked } = useAtomValue(limitOrdersAtom)
+
   return (
     <>
       <QuoteUpdater />
       <MarketPriceUpdater />
       <ActiveRateUpdater />
-      <styledEl.PageWrapper>
+      <styledEl.PageWrapper isUnlocked={isUnlocked}>
```

Comment:
@nenadV91 Unfortunately, I haven't found a solution to move the displaying responsibility inside the widget. If you have a better solution - I will be happy to you pushing it directly in this branch :)